test/acceptance/features/steps/steps.py

```diff
@@ -293,8 +330,8 @@ def create_cr(context, cr_name):
     openshift = Openshift()
     yaml = context.text
     output = openshift.oc_apply(yaml)
-    result = re.search(rf'.*{cr_name}.*(created|unchanged)', output)
-    result | should_not.be_none.desc("CRD {cr_name} Created")
+    result = re.search(rf'.*{cr_name}.*(created|unchanged|configured)', output)
+    result | should_not.be_none.desc("CR {cr_name} Created/Updated")
```

Comment:
I think variable used here is {crd_name} not {cr_name}. Also update CR to CRD.

Reply:
We are checking for CR here . Function says create_cr and takes in cr_name as the variable